e2e/transfer_test.go

```diff
@@ -208,6 +226,56 @@ func (s *TransferTestSuite) TestMsgTransfer_Timeout_Nonincentivized() {
 	})
 }
 
+// TestSendEnabledParam tests changing ics20 SendEnabled parameter
+func (s *TransferTestSuite) TestSendEnabledParam() {
+	t := s.T()
+	ctx := context.TODO()
+
+	_, channelA := s.SetupChainsRelayerAndChannel(ctx, transferChannelOptions())
+	chainA, chainB := s.GetChains()
+
+	chainADenom := chainA.Config().Denom
+
+	chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+	chainAAddress := chainAWallet.Bech32Address(chainA.Config().Bech32Prefix)
+
+	chainBWallet := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
+	chainBAddress := chainBWallet.Bech32Address(chainB.Config().Bech32Prefix)
+
+	s.Require().NoError(test.WaitForBlocks(ctx, 1, chainA, chainB), "failed to wait for blocks")
+
+	t.Run("ensure transfer sending is enabled", func(t *testing.T) {
+		enabled := s.QueryTransferSendEnabledParam(ctx, chainA)
+		s.Require().True(enabled)
+	})
+
+	t.Run("ensure packets can be sent", func(t *testing.T) {
+		transferTxResp, err := s.Transfer(ctx, chainA, chainAWallet, channelA.PortID, channelA.ChannelID, testvalues.DefaultTransferAmount(chainADenom), chainAAddress, chainBAddress, s.GetTimeoutHeight(ctx, chainB), 0)
+		s.Require().NoError(err)
+		s.AssertValidTxResponse(transferTxResp)
+	})
+
+	t.Run("change send enabled parameter to disabled ", func(t *testing.T) {
```

Comment:
[nit] whitespace at the end of test names makes the output have trailing `_`

e.g. `change_send_enabled_parameter_to_disabled_`

```suggestion
	t.Run("change send enabled parameter to disabled", func(t *testing.T) {
```